packages/compass/src/app/components/update-toasts.tsx

```diff
@@ -105,11 +105,14 @@ export function onAutoupdateStarted({ newVersion }: { newVersion: string }) {
     title: `Compass ${newVersion} is downloading`,
   });
 }
-export function onAutoupdateFailed() {
+export function onAutoupdateFailed(reason?: 'outdated-operating-system') {
   openToast(updateToastId, {
     variant: 'warning',
     title: 'Failed to download Compass update',
-    description: 'Downloading a newer Compass version failed',
+    description:
+      reason === 'outdated-operating-system'
+        ? 'The version of your operating system is no longer supported.'
```

Comment:
Maybe a link to the docs install page where we list minimal reqs? This one https://www.mongodb.com/docs/compass/current/install/

Reply:
I wonder if that would be clickable? 🤔

Reply:
We have both buttons and links in toasts in some places in Compass, if you render it as a link, it should be clickable